### DISCUSSION TITLE
Hi! Would you mind give me some common combo files?

### DISCUSSION THREAD

**Discussion by neroIcici** [DrPython3/MailRipV2#10] [Q&A]
Hi,
   MailRipV2 is very useful, I learned a lot from this program.
   And, i know it sounds a little bit bothering....
   but, would you mind give me a combo file just for test?
   THX!!!

**Comment by DrPython3** [comment]
Of course: no! If you do not have a list of email:pass combos, create one using your own email accounts. That should be enough for testing purposes.
